glacier_mapping/models/frame.py

```diff
@@ -49,7 +48,7 @@ def __init__(self, loss_fn=None, model_opts=None, optimizer_opts=None,
         optimizer_def = getattr(torch.optim, optimizer_opts.name)
         self.optimizer = optimizer_def(self.model.parameters(), **optimizer_opts.args)
         self.lrscheduler = ReduceLROnPlateau(self.optimizer, "min",
-                                             verbose=True, patience=500,
+                                             verbose=True, patience=10,
```

Comment:
Did you mean to use patience=10? This seems like a low number to be using.

Reply:
Yes, since we only train for 250 epochs max and the model diverges quickly with a high learning rate after a certain point, smaller patience made more sense. However, I haven't explored all the options here.

Reply:
Ah, okay I was still thinking of your long-running example. No need to explore different parameters here, I just wanted to make sure it wasn't a mistake.